src/uikit.cs

```diff
@@ -14957,6 +14964,7 @@ interface UIPrintInteractionController {
 		bool ShowsPaperSelectionForLoadedPapers { get; set; }
 
 		[iOS (8,0)]
+		[Async (ResultTypeName = "UIPrintInteractionResult")]
```

Comment:
I do not think we can Async'ify this one according to [docs](https://developer.xamarin.com/guides/cross-platform/macios/binding/binding-types-reference/#AsyncAttribute):

> You can use the `[Async]` attribute on methods that **return void** and whose **last** argument is a callback.